src/SourceMap.js

```diff
@@ -79,12 +79,14 @@ class SourceMap {
   }
 
   addMapping(mapping, lineOffset = 0, columnOffset = 0) {
-    mapping.generated = {
-      line: mapping.generated.line + lineOffset,
-      column: mapping.generated.column + columnOffset
-    };
-
-    this.mappings.push(mapping);
+    this.mappings.push({
```

Comment:
Not familiar with this API, but does this `mapping` have a `.name`? I see it used in `addConsumerMapping`, but not this one.

Reply:
Ow it should, thanks for noticing